homepage.css

```diff
@@ -0,0 +1,34 @@
+body{
+    background-color: var(--color-black-04);
+    color: var(--color-black-00);
+}
+
+.main-body{
+    width:80%;
+    margin: auto;
+}
+
+.container{
+    display:flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    gap:2rem;
+    margin-top:2rem;
+
+}
+/* Overwrite */
+.card {
+    padding: 0;
+    width: 25rem;
+    border-radius: 1rem;
+    text-decoration: none;
+}
+.card-media {
+    border-radius: 1rem;
+    height: 17.5rem;
+    object-fit: fill;
+}
+.card-body{
+    padding:0 1rem;
+    padding-bottom: 1rem;;
```

Comment:
Improvement: It might be you didn't notice. redundancy of padding.

Reply:
hey thanks for pointing out